ext/fs/interface.rs

```diff
@@ -361,3 +415,289 @@ fn string_from_utf8_lossy(buf: Vec<u8>) -> String {
     Cow::Borrowed(_) => unsafe { String::from_utf8_unchecked(buf) },
   }
 }
+
+// todo(dsherret): this is temporary. Instead of using the `FileSystem` trait implementation
+// in the CLI, the CLI should instead create it's own file system using `sys_traits` traits
+// then that can implement the `FileSystem` trait. Then this `FileSystem` trait can stay here
+// for use only for `ext/fs` and not the entire CLI.
+#[derive(Debug, Clone)]
```

Comment:
Do you have a follow up PR prepared or is this a TODO for some other time in the future? If the latter, consider opening an issue for it

Reply:
I have a PR incoming after this one.